src/aind_data_transfer/transformations/ome_zarr.py

```diff
@@ -570,3 +609,183 @@ def _populate_metrics(
     storage_ratio = _get_storage_ratio(out_zarr, tile_name)
     tile_metrics["storage_ratio"] = storage_ratio
     LOGGER.info(f"Compress ratio: {storage_ratio}")
+
+
+def _build_ome(
+    data_shape: Tuple[int, ...],
+    image_name: str,
+    channel_names: Optional[List[str]] = None,
+    channel_colors: Optional[List[int]] = None,
+    channel_minmax: Optional[List[Tuple[float, float]]] = None,
+) -> Dict:
+    """
+    Create the necessary metadata for an OME tiff image
+
+    Parameters
+    ----------
+    data_shape: A 5-d tuple, assumed to be TCZYX order
+    image_name: The name of the image
+    channel_names: The names for each channel
+    channel_colors: List of all channel colors
+    channel_minmax: List of all (min, max) pairs of channel intensities
+
+    Returns
+    -------
+    Dict: An "omero" metadata object suitable for writing to ome-zarr
+    """
+    if channel_names is None:
+        channel_names = [
+            f"Channel:{image_name}:{i}"
+            for i in range(data_shape[1])
+        ]
+    if channel_colors is None:
+        channel_colors = [i for i in range(data_shape[1])]
+    if channel_minmax is None:
+        channel_minmax = [(0.0, 1.0) for _ in range(data_shape[1])]
+    ch = []
+    for i in range(data_shape[1]):
+        ch.append(
+            {
+                "active": True,
+                "coefficient": 1,
+                "color": f"{channel_colors[i]:06x}",
+                "family": "linear",
+                "inverted": False,
+                "label": channel_names[i],
+                "window": {
+                    "end": float(channel_minmax[i][1]),
+                    "max": float(channel_minmax[i][1]),
+                    "min": float(channel_minmax[i][0]),
+                    "start": float(channel_minmax[i][0]),
+                },
+            }
+        )
+
+    omero = {
+        "id": 1,  # ID in OMERO
+        "name": image_name,  # Name as shown in the UI
+        "version": "0.4",  # Current version
```

Comment:
This should not be done right now, but we have to think about ways of managing the version. A class that creates the ome metadata based on documentation for the upcoming versions.

Reply:
I agree that would be useful to have. `0.4.1` is the latest version, but we only use features introduced in `0.4.0` at the moment. I don't foresee a need for backwards compatibility, so we could just keep everything updated to the latest version.